e2e/fee_middleware_test.go

```diff
@@ -18,13 +24,51 @@ type FeeMiddlewareTestSuite struct {
 	testsuite.E2ETestSuite
 }
 
+// RegisterCounterPartyPayee broadcasts a MsgRegisterCounterpartyPayee message.
+func (s *FeeMiddlewareTestSuite) RegisterCounterPartyPayee(ctx context.Context, chain *cosmos.CosmosChain, user broadcast.User, portID, channelID, relayerAddr, counterpartyPayeeAddr string) (sdk.TxResponse, error) {
```

Comment:
With so many arguments the line becomes quite long, should we split it in multiple lines for better readability?